### DISCUSSION TITLE
AIO Auto Update

### DISCUSSION THREAD

**Discussion by dng-nguyn** [nextcloud/all-in-one#4872] [Questions] [answered]
Hello,
Is it possible to enable auto update for aio containers without having to enable daily backup? I have a separate external backup solution for nextcloud, and I don't want to have 2 backups just to enable auto update.
Thanks,

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-enable-automatic-updates-without-creating-a-backup-beforehand
